src/annotator/config/index.js

```diff
@@ -192,21 +197,28 @@ const configDefinitions = {
 };
 
 /**
- * Return the configuration for a given application context.
+ * Return the subset of Hypothesis client configuration that is relevant in
+ * a particular context.
+ *
+ * See https://h.readthedocs.io/projects/client/en/latest/publishers/config/
+ * for details of all available configuration and the different ways they
+ * can be included on the page. In addition to the configuration provided by
+ * the embedder, the boot script also passes some additional configuration
+ * to the annotator, such as URLs of the various sub-applications and the
+ * boot script itself.
  *
- * @param {AppContext} [appContext] - The name of the app.
+ * @param {Context} context
  */
-export function getConfig(appContext = 'annotator', window_ = window) {
+export function getConfig(context, window_ = window) {
```

Comment:
I removed the default argument because I don't think it makes sense. The caller should always specify which configuration they need.